### PR TITLE
refact: fn utility updating referral reward

### PR DESCRIPTION
**Pull request by juniormp**
Issue #52

### CODE REVIEW

File: src/utils/webhook.js [hunk 1]
```diff
@@ -1272,6 +1189,50 @@ export async function handleNewTransaction(params) {
   return false;
 }
 
+async function updateReferralReward(
```
Comment (tcoratger) [inline src/utils/webhook.js:1192]: Can we create a separate file to put this function there. I would say:
- A util file for sign up rewards
- A util file for referral rewards
- A util file link rewards
- A util file for transactions between users

in order to keep stuffs clean an avoid overloading this "main" file.

Comment (juniormp) [reply src/utils/webhook.js:1174]: @tcoratger so it would be the same function for each file ? If yes we will end up with the same problem.

Comment (juniormp) [reply src/utils/webhook.js:1174]: I agree I think its better.
Decouple the support functions from the main file, and while making new features study new possibilities to re-organize the main file with support utils files.

Comment (juniormp) [reply src/utils/webhook.js]: @tcoratger I added the file `rewardHelpers.js` what do you think about this name ?

Comment (tcoratger) [reply src/utils/webhook.js]: @juniormp Seems good to me

File: src/utils/rewardHelpers.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+async function updateReferralReward(
+    db,
+    params
+  ) {
+    const REWARDS_COLLECTION = 'rewards';
```
Comment (tcoratger) [inline src/utils/rewardHelpers.js:5]: Can you use our global constant for that here?

File: src/utils/rewardHelpers.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+async function updateReferralReward(
```
Comment (tcoratger) [inline src/utils/rewardHelpers.js:1]: Can we change the name of the function here for `updateRewardDB` for example if we use it for all rewards?

File: src/utils/webhook.js [hunk 4]
```diff
@@ -223,22 +202,16 @@ export async function handleSignUpReward(db, params) {
...
-          $set: {
-            userTelegramID: params.userTelegramID,
-            eventId: params.eventId,
-            reason: 'user_sign_up',
-            status: TRANSACTION_STATUS.PENDING_HASH,
-            userOpHash: txReward.data.userOpHash,
-          },
-        },
-        { upsert: true }
+          //responsePath: responsePath,
```
Comment (tcoratger) [inline src/utils/webhook.js:211]: Can you remove this if unused?

**Pull request merged**